tests/sdk/test_circuit_v3.py

```diff
@@ -0,0 +1,52 @@
+from typing import Generator
+from unittest.mock import MagicMock
+
+import pytest
+from pytest_mock import MockerFixture
+
+from quantuminspire.sdk.models.circuit_v3 import CircuitV3
+
+MOCK_QUANTUM_CIRCUIT = "quantum circuit"
+
+
+@pytest.fixture
+def CircuitBuilder(mocker: MockerFixture) -> Generator[MagicMock, None, None]:
+    yield mocker.patch("quantuminspire.sdk.models.circuit_v3.CircuitBuilder")
+
+
+def test_get_program_name(CircuitBuilder: MagicMock) -> None:
+    with CircuitV3(platform_name="platform", program_name="program", number_of_qubits=2) as c:
+        pass
+
+    assert c.program_name == "program"
```

Comment:
Maybe we could write this as follows:
```
with CircuitV3(platform_name="platform", program_name="program", number_of_qubits=2) as c:
        assert c.program_name == "program"
```
This will save us a few lines but also makes sure we are using the `with` block as intended (`c` could be altered by `close()` calls after we refer to it outside the with-block)

Reply:
The `program_name` should actually stay the same after the context manager is closed. The program name is used when uploading the algorithm, which happens at the end of the script.